### DISCUSSION TITLE
Rename `blender` executable to `upblender`

### DISCUSSION THREAD

**Discussion by paul-marechal** [UPBGE/upbge#858] [Ideas/Features for UPBGE 0.3+]
It might sound silly, but this request comes from the fact that people will most certainly have BF Blender with `blender` and `blenderplayer` in their path.

To ease with the cohabiting, I propose to rename the executables to something unique, `upblender` and `upblenderplayer`.

What do you think?

**Comment by panzergame** [comment]
Personally I use `upbge` and `upbgeplayer` (and for debug `upbgedb` and `upbgedbplayer`).

**Comment by paul-marechal** [comment]
I like those names too, but do you mean that it should be the user's responsibility to rename them, and that it shouldn't be done directly when releasing?

**Comment by lordloki** [comment]
I think that will be done for next release (0.2.5) as part of UPBGEzation of Blender. 
We also have to replace icons for linux and macos and serveral tasks incompletes yet.

**Comment by sambler** [comment]
Having a plan to keep upbge as an onging project means that the renaming should get done. The question I want to add is how deep do you want/have to go with renaming?

- Rename installed binaries
- Rebrand splash screen/image
- Rename or add upbge file extensions
- Rename preference files/folders - use upbge version in path?
- Version numbers - keep current 0.2/0.3... or stay inline with blender versions?

While it is fine to fork and develop blenders code, the blender name is a trademark of the Blender Foundation and can't legally be used by forked projects.

**Comment by BluePrintRandom** [comment]
[winblender.zip](https://github.com/UPBGE/upbge/files/3909350/winblender.zip)
here is a icon the community made / voted that they liked for 0.3.0

**Comment by lordloki** [comment]
Almost all these ones were available in last 0.2.4 build ( own icon, references, etc). Before we release 0.3 I/We need to get upbge at 0.2.4 level (at this moment we are at 0.1.9 level). This will take time (1 year at least) until then we will be in alpha status.

**Comment by youle31** [comment]
I move this to discussions
